experimental/web/generate_web_metrics.sh

```diff
@@ -0,0 +1,199 @@
+###############################################################################
+
+wget -nc https://storage.googleapis.com/iree-model-artifacts/mobile_ssd_v2_float_coco.tflite
+wget -nc https://storage.googleapis.com/iree-model-artifacts/deeplabv3.tflite
+wget -nc https://storage.googleapis.com/iree-model-artifacts/posenet.tflite
+wget -nc https://storage.googleapis.com/iree-model-artifacts/mobilebert-baseline-tf2-float.tflite
+wget -nc https://storage.googleapis.com/iree-model-artifacts/mobilenet_v2_1.0_224.tflite
+wget -nc https://storage.googleapis.com/iree-model-artifacts/MobileNetV3SmallStaticBatch.tflite
+
+###############################################################################
+# Import programs into MLIR                                                   #
+###############################################################################
+
+# Note: you can also download imported programs from runs of the
+# https://buildkite.com/iree/iree-benchmark-android pipeline.
+
+IREE_IMPORT_TFLITE_PATH=iree-import-tflite
+
+# import_program helper
+#   Args: program_name, tflite_source_path
+#   Imports tflite_source_path to program_name.tflite.mlir
+function import_program {
+  OUTPUT_FILE=./$1.tflite.mlir
+  echo "Importing '$1' to '${OUTPUT_FILE}'..."
+  "${IREE_IMPORT_TFLITE_PATH?}" "$2" -o "${OUTPUT_FILE}"
+}
+
+import_program "deeplabv3" "deeplabv3.tflite"
+import_program "mobile_ssd_v2_float_coco" "mobile_ssd_v2_float_coco.tflite"
+import_program "posenet" "posenet.tflite"
+import_program "mobilebertsquad" "./mobilebert-baseline-tf2-float.tflite"
+import_program "mobilenet_v2_1.0_224" "./mobilenet_v2_1.0_224.tflite"
+import_program "MobileNetV3SmallStaticBatch" "MobileNetV3SmallStaticBatch.tflite"
+
+###############################################################################
+# Compile programs                                                            #
+###############################################################################
+
+# Either build from source (setting this path), or use from the python packages.
+# IREE_COMPILE_PATH=~/code/iree-build/iree/tools/iree-compile
+# IREE_COMPILE_PATH="D:\dev\projects\iree-build\iree\tools\iree-compile"
+IREE_COMPILE_PATH=iree-compile
+
+# compile_program_wasm helper
+#   Args: program_name
+#   Compiles program_name.tflite.mlir to program_name_wasm.vmfb, dumping
+#   statistics and intermediate files to disk.
+function compile_program_wasm {
+  INPUT_FILE=./"$1".tflite.mlir
+  OUTPUT_FILE=./"$1"_wasm.vmfb
+  echo "Compiling '${INPUT_FILE}' to '${OUTPUT_FILE}'..."
+
+  ARTIFACTS_DIR=./"$1"-wasm_artifacts/
+  mkdir -p "${ARTIFACTS_DIR}"
+
+  # Compile from .mlir to .vmfb, dumping all the intermediate files and
+  # compile-time statistics that we can.
+  "${IREE_COMPILE_PATH?}" "${INPUT_FILE}" \
+    --iree-input-type=tosa \
+    --iree-hal-target-backends=llvm \
+    --iree-llvm-target-triple=wasm32-unknown-emscripten \
```

Comment:
I'll be extending these flags in the future to include a few variants like
`--iree-llvm-target-cpu-features=+atomics,+bulk-memory,+simd128` (~2x speedup from some initial experiments)